faq/tidb-faq.md

```diff
@@ -83,6 +83,18 @@ Currently [TiDB documentation](/overview.md#tidb-introduction) is the most impor
 
 32 characters at most.
 
+### What are the limits on the column number and row size in TiDB?
+
+- The maximum number of columns TiDB defaults to 1017. You can adjust the number up to 4096.
+- The maximum size of a single row defaults to 6 MB. You can increase the number up to 120 MB.
+
+For more information, see [TiDB Limitations](/tidb-limitations.md).
+
+### Does TiDB support JSON or protocal buffers (protobuf)?
```

Comment:
Shall we remove this question for a while as JSON is GA after S10 (near September end)?   We will bring it back after S10 is released.

Reply:
OK. Removed for now.